build/Build.Steps.cs

```diff
@@ -129,7 +131,7 @@ partial class Build
 
             DotNetMSBuild(x => x
```

Comment:
DotNetMSBuild is build in the Nuke itself, so I was able to remove our custom class.